myproject/blog/views.py

```diff
@@ -21,14 +24,14 @@ class BlogList(generic.ListView):
      def get_queryset(self):
          return BlogPost.objects.order_by("-created_at")
     
-class AuthorProfile(generic.DetailView):
+class BloggerDetail(generic.DetailView):
     model = Author
     template_name = "blog/author_detail.html"
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         author = self.get_object()
-        context['author_posts'] = author.blogs.all().order_by("-created_at")
+        context['author_posts'] = author.blogs.all()
```

Comment:
Tell me how many queries are executed for here? Attach the screenshot here not in the skype with the sql queries shown on the django debug toolbar.

Reply:
![Uploading Screenshot 2024-03-29 at 12.09.06 PM.png…]()